Dockerfile.backend

```diff
@@ -1,10 +1,20 @@
-FROM node:18 as builder
 ARG ARCH=x86_64
 
-COPY . /src/
-RUN cd /src/ && \
-  yarn install && \
-  yarn build:backend
+FROM node:18 as pruner
+WORKDIR /src/
+COPY . .
+RUN yarn global add turbo
+RUN yarn prune:backend
+# RUN turbo prune --docker @l2beat/backend
```

Comment:
I guess we should either remove this comment or uncomment it and remove the prune:backend script. It's so specific to docker it won't ever be used anywhere else (for purposes other than debugging docker issues).

Reply:
makes sense, also comment was left here unintentionally. Fixed